examples/event_handling/legend_picking.py

```diff
@@ -5,45 +5,39 @@
 
 Enable picking on the legend to toggle the original line on and off
 """
+
 import numpy as np
 import matplotlib.pyplot as plt
 
-t = np.arange(0.0, 0.2, 0.1)
-y1 = 2*np.sin(2*np.pi*t)
-y2 = 4*np.sin(2*np.pi*2*t)
+
+t = np.linspace(0, 1)
+y1 = 2 * np.sin(2*np.pi*t)
+y2 = 4 * np.sin(2*np.pi*2*t)
 
 fig, ax = plt.subplots()
 ax.set_title('Click on legend line to toggle line on/off')
-line1, = ax.plot(t, y1, lw=2, label='1 HZ')
-line2, = ax.plot(t, y2, lw=2, label='2 HZ')
-leg = ax.legend(loc='upper left', fancybox=True, shadow=True)
-leg.get_frame().set_alpha(0.4)
+line1, = ax.plot(t, y1, lw=2, label='1 Hz')
+line2, = ax.plot(t, y2, lw=2, label='2 Hz')
+leg = ax.legend(fancybox=True, shadow=True)
 
-
-# we will set up a dict mapping legend line to orig line, and enable
-# picking on the legend line
 lines = [line1, line2]
-lined = dict()
+lined = {}  # Will legend lines to original lines.
 for legline, origline in zip(leg.get_lines(), lines):
-    legline.set_picker(5)  # 5 pts tolerance
+    legline.set_picker(True)  # Enable picking on the legend line.
```

Comment:
Shouldn't you *also*, keep `legline.set_pickradius(5)`? Otherwise you'd need to hit the line exactly,I think (not tested).

Reply:
No, 5 is the default pickradius.